pkg/components/imguploader/gcsuploader.go

```diff
@@ -20,19 +20,21 @@ const (
 type GCSUploader struct {
 	keyFile string
 	bucket  string
+	path	string
 	log     log.Logger
 }
 
-func NewGCSUploader(keyFile, bucket string) *GCSUploader {
+func NewGCSUploader(keyFile, bucket, path string) *GCSUploader {
 	return &GCSUploader{
 		keyFile: keyFile,
 		bucket:  bucket,
+		path:	 path,
 		log:     log.New("gcsuploader"),
 	}
 }
 
 func (u *GCSUploader) Upload(ctx context.Context, imageDiskPath string) (string, error) {
-	key := util.GetRandomString(20) + ".png"
+	key := u.path + util.GetRandomString(20) + ".png"
```

Comment:
Use `path.Join` instead https://golang.org/pkg/path/#Join

Reply:
Ok, Thank you.